tests/coverage/async.rs

```diff
@@ -86,6 +86,14 @@ fn l(x: u8) {
 
 async fn m(x: u8) -> u8 { x - 1 }
 
+async fn n() -> u8 { 1 }
+
+async fn o() -> u8 {
+    // await should be covered even if the function never yields
+    n()
+        .await
+}
+
```

Comment:
You don't necessarily *have* to do this, but it's probably a good idea to put this in its own new test, rather than adding more stuff to `async.rs`.

Reply:
(If you do this, I will probably have some more feedback about how to set up the module attributes and `//@` headers, but it's probably easier for me to see what you've written and then tell you what to change.)